src/components/utilities/dialog.tsx

```diff
@@ -1,8 +1,13 @@
 import { inject, observer } from "mobx-react";
-import React from "react";
+import React, { CSSProperties } from "react";
 import { BaseComponent, IBaseProps } from "../base";
 import { UIDialogModelType } from "../../models/stores/ui";
 
+import CloseIcon from "../../assets/icons/close/close.svg";
+import CopyToDocumentIcon from "../../clue/assets/icons/copy-to-document-tool.svg";
+import AlertIcon from "../../assets/alert-icon.svg";
+import ConfirmIcon from "../../assets/confirm-icon.svg";
+
 import "./dialog.sass";
```

Comment:
A comment explaining what the intention of this `DialogComponent` is would be welcome here.
Is the idea that over time most dialogs should be making use of this component?

Reply:
Originally when I setup CLUE all the dialog where supposed to go through this component but at some point folks started building separate dialogs.  I didn't realize this until I added the copy to document dialog.  I'm not sure if that should be the pattern moving forward but at least we got the standard "built-in" dialog styles updated :)